templates/lookup_plugins/osinfo.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
```

Comment:
Please add a a description to this script.

What does it do, why and what is it used for?

Reply:
Copyright?

Reply:
Sure, but what license do we want to use?